### ISSUE TITLE
[2018] Missing Images

### ISSUE THREAD

**Issue by BBlackwo** [bestofjs/javascript-risingstars#34]
These are all the missing images I found. You can see it's showing the alt text on the ToC.

<img width="686" alt="screen shot 2019-01-05 at 2 17 42 pm" src="https://user-images.githubusercontent.com/7598058/50719729-ef0a3900-10f4-11e9-8548-2196eab2e665.png">

- Vue Element Admin
- Hyperapp
- Omi
- Next.Js
- React Native
- Docz
- Bulma
- Formik
- Hyperapp
- Mithril
- Sails
- Loopback
- Docsarus

**Comment by michaelrambeau** [comment]
That's strange, thank you for the screenshot.

It looks like a browser cache or security issue because the images you can't see are the ones that are hosted on `https://bestofjs.org`.

For example: https://bestofjs.org/logos/hyperapp.png for Hyperapp.

Could you try to open the site in a private browser window ("New private window" on Firefox) or ("New Incognito Window" or Chrome)? And can you try in an other browser?

In the past, I had similar issue with the application `https://bestofjs.org/`, sometimes the latest assets are not served correctly...

**Comment by BBlackwo** [comment]
I think it's some sort of load balancing / multiple servers issue. If I open https://bestofjs.org/logos/hyperapp.png in incognito mode several times (e.g. 5 times), sometimes it works (4 times), sometimes it doesn't (1 time). 

Normal chrome browsing does this (so it's probably stuck on the server without the image):

<img width="514" alt="screen shot 2019-01-05 at 5 59 27 pm" src="https://user-images.githubusercontent.com/7598058/50721448-0278cc80-1114-11e9-887e-fa2b6057a11d.png">

But sometimes in incognito it works some of the time:

<img width="535" alt="screen shot 2019-01-05 at 5 59 06 pm" src="https://user-images.githubusercontent.com/7598058/50721456-0efd2500-1114-11e9-8426-ceb5ec37f9ef.png">

**Comment by michaelrambeau** [comment]
Maybe it has to do with the cache on some instances of the load balancing system, because I guess that if you try an **older** logo like this one:

https://bestofjs.org/logos/angular.svg

...it always works, in all browsers, right, can you try?

Angular logo has been uploaded 8 months ago, while the Hyperapp logo is more recent, that's the difference.

**Comment by BBlackwo** [comment]
Yeah I think you're right. That Angular logo works fine every time in all browsers and I tried 6-7 times in incognito.

**Comment by michaelrambeau** [comment]
OK thank you very much for all the tests.

In the past, the only way I found to solve the problem was to build _Best of JS_ from an other branch, different from `master`...  that's what I have just done. Doing this seems to somehow invalidate the cache at the CDN servers level used by Netlify (Netlify is the service used to host all my _Best of JS_ apps).

Could you try again the failing test about the Hyperapp logo?

**Comment by michaelrambeau** [comment]
...actually it does not seem to work (or not yet) because I have the same problem as you: opening the `Hyperapp` logo URL in a new Private window does not return the image... I'm doing more more tests, sorry for all my requests!

**Comment by BBlackwo** [comment]
Hmm okay. Yeah you should be able to reproduce it the same way I was. Let me know if you need anything else. Cheers.

**Comment by michaelrambeau** [comment]
Hello Benjamin @BBlackwo , I think I've found the cause of the problem.

Some months ago, I enabled a feature called "Split Testing" on the Netlify application, users were redirected randomly to a feature branch (that is now old). I totally forgot about that, it's my bad.

![image](https://user-images.githubusercontent.com/5546996/50730749-d56dfd80-1197-11e9-8518-2577ea37b448.png)


Could you try again the tests we run yesterday?

Thank you again for your help!

**Comment by BBlackwo** [comment]
@michaelrambeau works fine now. Good find :+1:

No worries for the help. Glad I mentioned it to you cause you might not have found it as your browser was hitting the correct branch 😄
